server/controllers/ArticleController.js

```diff
@@ -1,18 +1,18 @@
 /**
   * This class contains all the methods responsible for creating and querying
-  * user profiles on the app
+  * articles on the app
   * It is made up static methods which can be called from anywhere in the app.
   */
 export default class ArticlesController {
```

Comment:
I noticed you changed the name of the file. Well done. However, your class name is still pluralised. Please review and make the necessary change as soon as you can.